pulp_container/tests/functional/utils.py

```diff
@@ -52,18 +52,20 @@
 
 CREATE_USER_CMD = [
     "from django.contrib.auth import get_user_model",
-    "from django.contrib.auth.models import Permission",
+    "from guardian.shortcuts import assign_perm",
+    "",
+    "from pulp_container.app.models import ContainerDistribution",
     "",
     "user = get_user_model().objects.create(username='{username}')",
     "user.set_password('{password}')",
     "user.save()",
-    "for permission in {permissions!r}:",
-    "    if '.' in permission:",
-    "        app_label, codename = permission.split('.', maxsplit=1)",
-    "        perm = Permission.objects.get(codename=codename, content_type__app_label=app_label)",
-    "    else:",
-    "        perm = Permission.objects.get(codename=permission)",
-    "    user.user_permissions.add(perm)",
+    "",
+    "for permission in {model_permissions!r}:",
+    "    assign_perm(permission, user)",
+    "",
+    "for permission, obj_name in {object_permissions!r}:",
+    "    obj = ContainerDistribution.objects.filter(name=obj_name)",
```

Comment:
Can't we have a more general way to address objects?
I tried to design this piece of code in a way, that we can move it to pulp-smash at some time, but the dependence on specific container models would prevent that.

Now that I am writing this, i realize, that we also do not follow that goal with the way we provide the Api's here. This might need more thought.